.github/hs-deploy-action/undeploy-hs.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+echo "-----"
+echo "Undeploying issue ${GITHUB_HEAD_REF}"
+
+ISSUE_NO=`echo "${GITHUB_HEAD_REF}" | cut -d'-' -f 2`
+upp application undeploy -b spartacus-feature-spa${ISSUE_NO} -e stage
```

Comment:
I think it should also be updated to the new id

Reply:
thanks. Good catch

Reply:
Done.